modules/compliance-anatomy.adoc

```diff
@@ -232,6 +252,11 @@ The remediations the `mc` currently consists of are listed in the machine config
 [source,terminal]
```

Comment:
The `$ oc get mc | grep 75-` command needs to be separated from the example output.